ovirt/resource_datecenter.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2017 Battelle Memorial Institute
+// All rights reserved.
+//
+// This software may be modified and distributed under the terms
+// of the BSD-2 license.  See the LICENSE file for details.
+
+package ovirt
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	ovirtsdk4 "gopkg.in/imjoey/go-ovirt.v4"
+)
+
+func resourceDataCenter() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDataCenterCreate,
+		Read:   resourceDataCenterRead,
+		Delete: resourceDataCenterDelete,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
The attribute `description`  is not required for creating a datacenter. So the `Required` should be `false`.